poedit.appdata.xml

```diff
@@ -1,31 +1,76 @@
 <?xml version="1.0" encoding="utf-8"?>
-<components version="0.11">
-  <component type="desktop-application">
+<component type="desktop">
     <id>poedit.desktop</id>
     <name>Poedit</name>
     <summary>Translations editor for gettext</summary>
     <project_license>MIT</project_license>
+    <metadata_license>CC0</metadata_license>
     <description>
-      <p>
-        Translations editor for gettext (PO files). It helps with translating
-        applications or WordPress themes and plugins into other languages, as
-        well as with developing localizable applications.
-      </p>
+        <p>
+            Translations editor for gettext (PO files). It helps with translating
+            applications or WordPress themes and plugins into other languages, as
+            well as with developing localizable applications.
+        </p>
     </description>
     <screenshots>
-      <screenshot>
-        <image type="source">https://upload.wikimedia.org/wikipedia/commons/c/c2/Poedit_1.8.1_en.png</image>
-      </screenshot>
+        <screenshot>
+            <image type="source">https://upload.wikimedia.org/wikipedia/commons/c/c2/Poedit_1.8.1_en.png</image>
+        </screenshot>
     </screenshots>
     <url type="homepage">https://poedit.net</url>
+    <url type="bugtracker">https://github.com/vslavik/poedit</url>
     <url type="translate">https://crowdin.com/project/poedit</url>
+    <content_rating type="oars-1.0" />
```

Comment:
Poedit doesn't contain any content of its own, so adding this tag, especially empty, doesn't seem to make sense to me. Per [the spec](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-content_rating) and contrary to this PR's description, `<content_rating>` is optional and not required, so why include it? Am I missing something?

Reply:
@vslavik It's not required for the spec but it's required for Flathub.

Reply:
Sounds like a Flathub bug to me, then — because such requirement is Majorly Stupid™ as explained above. And if this code, i.e. *value-less* tag, satisfies Flathub, that's doubly so :( But OK, so be it. An explanation that this is to keep Flathub happy would be nice, though.

Reply:
I'm not related to Flathub at all. Just a random geek trying to make software's easily installable on Linux :)
@nedrichards Do you have anything to say about this? Thanks